src/node/BrowserFetcher.ts

```diff
@@ -112,10 +112,14 @@ function downloadURL(
 function handleArm64(): void {
   fs.stat('/usr/bin/chromium-browser', function (err, stats) {
     if (stats === undefined) {
-      console.error(`The chromium binary is not available for arm64: `);
-      console.error(`If you are on Ubuntu, you can install with: `);
-      console.error(`\n apt-get install chromium-browser\n`);
-      throw new Error();
+      fs.stat('/usr/bin/chromium', function (err, stats) {
```

Comment:
(I'm not familiar with arm64) why would it maybe be in chromium-browser or chromium? Should we update the error message too?

Reply:
@jackfranklin The error was updated